tests/test_predictors.py

```
@@ -51,18 +51,15 @@
-    assert sorted(actual) == sorted(expected)
+    assert sorted(agg.labels, key=str) == sorted(expected_labels, key=str)
 
 
-def _get_centroid_for(agg, label_group):
-    """Look up the centroid vector for a given label group."""
-    key = tuple(sorted(label_group))
-    for i, g in enumerate(agg.labels):
-        if tuple(sorted(g)) == key:
+def _get_centroid_for(agg, label):
+    """Look up the centroid vector for a given label."""
+    for i, l in enumerate(agg.labels):
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)